nucypher/blockchain/eth/actors.py

```diff
@@ -858,10 +862,35 @@ def locked_tokens(self, periods: int = 0) -> NU:
 
     @property
     def current_stake(self) -> NU:
+        """The total number of staked tokens, i.e., tokens locked in the current period."""
+        return self.locked_tokens(periods=0)
+
+    @property
+    def active_stakes(self) -> Iterable[Stake]:
+        """Returns a list of active stakes for this staker."""
```

Comment:
Since not actually returning a List:

```suggestion
        """Returns active stakes for this staker."""
```

Reply:
:heavy_check_mark: